modelscope/msdatasets/utils/hf_datasets_util.py

```diff
@@ -41,8 +41,7 @@
                                        _MODULE_TO_EXTENSIONS,
                                        _PACKAGED_DATASETS_MODULES)
 from datasets.utils import file_utils
-from datasets.utils.file_utils import (OfflineModeIsEnabled,
```

Comment:
Cant del `OfflineModeIsEnabled` due to the usage of dataset_module_factory function.